azext_edge/edge/providers/check/base/display.py

```diff
@@ -0,0 +1,145 @@
+# coding=utf-8
+# ----------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License file in the project root for license information.
+# ----------------------------------------------------------------------------------------------
+
+from typing import Any, Dict, List, Optional, Tuple
+
```

Comment:
I feel like the functions under this file is basically used for check manager, what do you think of combining this file with the check_manager.py?

Reply:
I want to keep the check manager class separate here from the display functionality